plugins/vlc-video/vlc-video-source.c

```diff
@@ -394,12 +394,97 @@ static int vlcs_audio_setup(void **p_data, char *format, unsigned *rate,
 	return 0;
 }
 
+static inline void vlc_split_media_path(const struct dstr *new_path,
+	struct dstr *path_only, struct dstr *options)
+{
+	int i;
+	const char* path = new_path->array;
```

Comment:
Style:  The asterisk should be next to the variable name when declaring a pointer.  Use "const char *path".